modules/virt-creating-a-service-backed-by-virtual-machines.adoc

```diff
@@ -0,0 +1,142 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc
+
+[id="virt-creating-a-service-backed-by-virtual-machines_{context}"]
+
+= Creating a service backed by virtual machines
+
+To create a service backed by a virtual machine, you start the `VirtualMachineInstance`,
+if it is not already running. Once the instance is running, you connect to the instance, and
```

Comment:
Why do you need to connect to the instance ? (I don't think you do).

Reply:
Changed to: 

To create a service backed by a virtual machine, start the `VirtualMachineInstance`,
if it is not already running. Once the instance is running, create a service 
object to expose the instance.